lib/videojs-playlist-ui.js

```diff
@@ -66,6 +66,7 @@ const createThumbnail = function(thumbnail) {
     // simple thumbnails
     let img = document.createElement('img');
     img.src = thumbnail;
+    img.alt = '';
```

Comment:
This is coming from [a thread in the bcplayerdev Google Group](https://groups.google.com/forum/#!topic/bcplayerdev/JNf2HUWdlkc). I think `''` makes sense because we have the video title immediately following the image.